its/projects/Telemetry/Telemetry/Telemetry.csproj

```
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net9.0</TargetFramework>
```

I'm curious what the TFM of this entry looks like
```suggestion
    <TargetFramework>net9.0-windows</TargetFramework>
```

https://learn.microsoft.com/en-us/dotnet/standard/frameworks